python/pyarrow/tests/test_fs.py

```diff
@@ -15,24 +15,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from datetime import datetime, timezone, timedelta
 import gzip
 import os
 import pathlib
 import pickle
-
-import pytest
 import weakref
+from datetime import datetime, timedelta, timezone
 
 import pyarrow as pa
+import pytest
```

Comment:
Let's not mix PyArrow imports and third-party imports, can you move this upwards?